receiver/vcenterreceiver/scraper.go

```diff
@@ -262,6 +266,22 @@ func (v *vcenterMetricScraper) scrapeHosts(ctx context.Context, dc *mo.Datacente
 		errs.AddPartial(1, fmt.Errorf("failed to retrieve perf metrics for HostSystems: %w", err))
 		return
 	}
+
+	// Get all host vSAN metrics and store for later retrieval
+	clusterRefs := []*types.ManagedObjectReference{}
+	for _, compute := range v.scrapeData.computesByRef {
```

Comment:
We could make a little helper function for this so it's not duplicated code.

Reply:
In my new PRs this doesn't exist as we don't have to worry about passing in cluster anymore

Reply:
resolve